README-ja.md

```diff
@@ -0,0 +1,32 @@
+# イントロダクション
+Xpring SDKは、Xpringとのインタラクションを簡単かつ直感的に行えるように設計されたライブラリとサービスのセットです。   Xpring SDKは、言語、プラットフォーム間で一貫して動作し、アプリを構築する言語、プラットフォーム、または環境に関係なく、凝集性のある開発エクスペリエンスを作成します。
+
+## アーキテクチャ
+開発エクスペリエンスは二つのコンポーネントで構成されています:
+- 言語固有のクライアント側ライブラリのセット
+- クライアント側ライブラリからのリクエストを処理し、それらをXRPノードにプロキシするサーバー側コンポーネント
+
+### クライアント側ライブラリ
+Xpring SDKは、XRP Ledgerで最も一般的に使用される機能を提供します。機能とプログラミングのパラダイムは、Xpring SDKのすべてのライブラリに共通です。
+
+Xpring SDKは以下の機能を提供します:
+- ウォレットの生成と派生（シードベースまたはHDウォレットベース）
+- アドレス検証
+- 口座残高の取得
+- XRP支払いの送信
```

Comment:
@U-top-Yoshida , this is _minor_, but maybe make this list the same as the one that opens the Xpring JS doc? 

https://github.com/xpring-eng/Xpring-JS/pull/80/files#diff-4c15cb023d73eb32f305098131af8fe9R10

> Xpring SDKは次の機能を提供します:
> - ウォレットの生成と派生（シードまたはHDウォレットベース）
> - アドレス検証
> - アカウントバランスの取得
> - XRP支払いの送信

Reply:
@gbarr01 Thank you for the comment, I will edit it.

Reply:
@gbarr01 I Edited the JS side. SDK translation will stay the same. Thank you.

Reply:
Hi @gbarr01 @U-top-Yoshida, are we waiting for any further changes here? Or can we merge this?